src/lxc/rexec.c

```diff
@@ -23,25 +23,27 @@
 #define LXC_MEMFD_REXEC_SEALS \
 	(F_SEAL_SEAL | F_SEAL_SHRINK | F_SEAL_GROW | F_SEAL_WRITE)
 
-static int push_vargs(char *data, int data_length, char ***output)
+static int push_vargs(const char *data, int data_length, char ***output)
```

Comment:
Please write a clear, concise comment describing what @output will look like on return.

Looks like it will be a set of N pointers into output itself, followed by a NULL pointer, followed by the original @data.

Reply:
Updated